tensorflow/contrib/layers/python/layers/layers.py

```diff
@@ -2563,7 +2563,7 @@ def separable_convolution2d(
           regularizer=weights_regularizer,
           trainable=trainable,
           collections=weights_collections)
-      strides = [1, stride_h, stride_w, 1]
+      strides = [1, 1, stride_h, stride_w] if data_format.startswith('NC') else [1, stride_h, stride_w, 1]
```

Comment:
Can you add a test where this change is tested? i.e. when num_outputs = None but depth_multiplier is Not.

Reply:
@sguada As far as I am aware, my test already does this check (`testConvNCHW` in `layers_test.py`), which is a part of this PR. Specifically, the test will assert correct output shape size for NCHW data input twice: Once with `num_outputs = None`, and with `num_outputs=5`, a non-None test value. Both tests are done with `depth_multiplier = 2`, which I believe satisfies the needed tests for this change. Let me know if I have missed something!